client/containers/Pub/PubDocument/PubInlineMenu.tsx

```diff
@@ -24,7 +25,22 @@ const PubInlineMenu = () => {
 	const { canView, canCreateDiscussions } = scopeData.activePermissions;
 	const selection = collabData.editorChangeObject!.selection;
 	const shouldHide = useMemo(() => {
+		const selectionInSuggestionRange = (): boolean => {
+			if (!collabData.editorChangeObject || !collabData.editorChangeObject.view) return false;
+			const doc = collabData.editorChangeObject.view.state.doc;
+			let inSuggestionRange = false;
+			doc.nodesBetween(selection.$anchor.pos - 1, selection.$head.pos + 1, (node) => {
+				if (inSuggestionRange) return;
+				const present = getSuggestionAttrsForNode(node);
+				if (present) inSuggestionRange = true;
+			});
+			return inSuggestionRange;
+		};
+
+		const inRange = selectionInSuggestionRange();
```

Comment:
I don't think you need the nodesBetween approach here either, and this could look identical to your function in the SE menu